.github/workflows/ci.yml

```diff
@@ -24,60 +24,64 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v4
       - name: Select Xcode Version
-        run: sudo xcode-select --switch /Applications/Xcode_15.3.app/Contents/Developer
+        run: sudo xcode-select --switch /Applications/Xcode_16.app/Contents/Developer
       - name: Download visionOS
         if: matrix.platforms == 'generic/platform=visionos'
-        run: xcodebuild -downloadPlatform visionOS
+        run: |
+          sudo xcodebuild -runFirstLaunch
+          sudo xcrun simctl list
+          sudo xcodebuild -downloadPlatform visionOS
+          sudo xcodebuild -runFirstLaunch
```

Comment:
Per https://github.com/actions/runner-images/issues/10559 what we had before was incomplete.